makefile

```diff
@@ -0,0 +1,24 @@
+# By default, this makefile will run everything inside a Docker container.
+# If you're a developer and want to run everything with your local `cargo`:
+# - Creates a `.env` file in the same folder as this `makefile`
+# - Add a line in `.env` with `LOCAL_DEV=true`
+-include .env
+
+ifeq ($(LOCAL_DEV),true)
+	SH=sh -c
+else
+	SH=docker run --rm --user "$(id -u)":"$(id -g)" --volume "${PWD}":/tmp/skip_error --workdir /tmp/skip_error rust:latest sh -c
```

Comment:
put a generic name (instead of skip_error)

Reply:
I renamed it `workspace`.